index.html

```diff
@@ -0,0 +1,197 @@
+          >
+            <i class="fa-solid fa-pen-to-square fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Autographs</h3>
+            <p class="p-1">
+              All your favorite authors will gather to make your dream a
+              reality.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-handshake fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">
+              Cosplayer meet&greet
+            </h3>
+            <p class="p-1">
+              Every big cosplayers from all over the world will be there.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-sharp fa-solid fa-star fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Special events</h3>
+            <p class="p-1">
+              The "gear fourth" show will be the wild card but more special
+              events will also take place.
+            </p>
+          </div>
+        </div>
+        <button type="button"
+          class="btn btn-danger p-3 rounded-0 text-white fw-bold mx-auto text-uppercase"
+        >
+          Claim your badge
+        </button>
+      </section>
+      <section id="guest" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 mb-3">Our virtual guests</h2>
+        <span class="line-primary"></span>
+        <ul class="row mt-5 list-unstyled p-3 justify-content-md-between"></ul>
+        <button type="button"
+          class="btn btn-danger p-3 rounded-0 text-white fw-bold mx-auto text-uppercase d-md-none more-or-less"
+        >
+          Load more
+        </button>
+      </section>
+      <section id="sponsor" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 mb-3 text-light">Sponsors</h2>
+        <span class="line-primary"></span>
+        <div class="brand w-75 d-flex justify-content-between my-5">
+          <i class="fab fa-xbox fs-1"></i>
+          <i class="fab fa-steam fs-1"></i>
+          <i class="fab fa-playstation fs-1"></i>
+          <i class="fab fa-imdb fs-1"></i>
+          <i class="fab fa-vimeo-v fs-1"></i>
+          <i class="fab fa-weibo fs-1"></i>
+        </div>
+      </section>
+    </main>
+    <footer
+      class="d-flex justify-content-around align-items-center my-4 w-50 mx-auto"
+    >
+      <img
+        src="img/logo/kohai-logo-transparent-bg.png"
+        alt="kohai logo"
+        class="logo"
+      />
+      <small class="fw-semibold">
+        2023 Unicorn Con <br />
+        Some rights Reserved</small
+      >
+    </footer>
```

Comment:
The about page of the desktop version has a dark background with no partner section as shown below. Kindly check and fix it. 

Your design 

![Screenshot from 2023-01-12 08-20-14](https://user-images.githubusercontent.com/65233860/212014651-570ecb55-4cd8-4407-8e6d-5d5e8eeaf05d.png)


Actual design 

![Screenshot from 2023-01-12 08-20-05](https://user-images.githubusercontent.com/65233860/212014713-19dff525-fab8-427a-89ba-19f6f0d47003.png)